plugin/src/automation/lift/lift.ml

```diff
@@ -536,7 +544,7 @@ let lift_core env c trm sigma =
          let sigma, args'' = map_rec_args lift_rec en sigma c args' in
          sigma, mkApp (f', args'')
        else
-         sigma, constr_app
+         sigma, constr_app (* TODO ... *)
```

Comment:
We may be able to use the normal constr rule here if we teach it to treat certain functions as opaque like a constructor, and get their arguments. But kind of risky